libraries/protocol/include/steem/protocol/config.hpp

```diff
@@ -33,12 +32,14 @@
 #define STEEM_ACCOUNT_RECOVERY_REQUEST_EXPIRATION_PERIOD  fc::seconds(12)
 #define STEEM_OWNER_UPDATE_LIMIT                          fc::seconds(0)
 #define STEEM_OWNER_AUTH_HISTORY_TRACKING_START_BLOCK_NUM 1
+
+#define STEEM_CHAIN_ID_NAME "testnet"
```

Comment:
Don't worry about the seed for the chain id. By definition, the chain ID is a sha256 hash. The cli option should just be a hash.